code/src/App.js

```diff
@@ -1,9 +1,83 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+
+import FormContent from './components/FormContent';
+import ListThoughts from './components/ListThoughts';
+
+import { API_URL } from 'reusable/url.js';
+import { API_URL_HEARTS } from 'reusable/url.js';
 
 export const App = () => {
+  const [thoughtsList, setThoughtsList]=useState([]);
+  const [newThought, setNewThought]=useState("");
+  const [thoughtLength, setThoughtLength] = useState('0');
+  
+  useEffect(() => {
+    fetchThoughtsList();
+  }, []);
+
+  const fetchThoughtsList = () => {
+    fetch(API_URL)
+      .then(response => response.json())
+      .then(thoughts => setThoughtsList(thoughts))
+      .catch(err => console.error(err));
```

Comment:
You need to remove all the console outputs before you send your code to production.